projects/optic/src/commands/run.ts

```diff
@@ -124,25 +124,28 @@ then posts a comment with a report to your PR/MR and exits with code 1 when issu
     )
     .configureHelp({ commandUsage: usage })
     .option(
-      '--ignore <ignore-glob>',
-      'OpenAPI specification files to ignore, comma separated globs.'
+      '-i, --ignore <glob_pattern,...>',
+      'Glob patterns matching specifications to ignore'
     )
-    .option(
-      '--include-git-ignored',
-      'Set to true to also match Git ignored files.',
-      false
+    .addOption(
+      new Option(
+        '-I, --include-git-ignored <bool>',
+        'Include specifications matched in .gitignore'
+      )
+        .choices(['true', 'false'])
+        .default('false')
     )
     .addOption(
       new Option(
-        '--severity <severity>',
-        'Set to "none" to prevent Optic from exiting 1 when issues are found.'
+        '-s, --severity <severity>',
+        'Control the exit code when there are issues: error=1, none=0'
```

Comment:
kind of hate the `error=1...` bit, but it seems like the most explicit way to just say what the option values actually represent.